test/dd/test_package.cpp

```diff
@@ -607,15 +617,15 @@ TEST(DDPackageTest, GarbageMatrix) {
   dd->incRef(bellMatrix);
   reducedBellMatrix =
       dd->reduceGarbage(bellMatrix, {false, true, false, false});
-  auto mat = reducedBellMatrix.getMatrix();
+  auto mat = dd->getMatrix(reducedBellMatrix, 2);
```

Comment:
This is calling a method that should no longer exist since it has been moved to the Edge.hpp file